src/core/Instrument.cpp

```diff
@@ -185,7 +186,7 @@ void Instrument::applyRelease( SampleFrame* buf, const NotePlayHandle * _n )
 	const auto releaseFrames = desiredReleaseFrames();
 
 	const auto endFrame = _n->framesLeft();
-	const auto startFrame = std::max(0, endFrame - releaseFrames);
+	const auto startFrame = std::max(0, static_cast<int>(endFrame) - static_cast<int>(releaseFrames));
```

Comment:
How about this?
```suggestion
	const auto startFrame = endFrame - std::min(endFrame, releaseFrames);
```
This avoids the unsigned subtraction problem while maintaining appropriate types.